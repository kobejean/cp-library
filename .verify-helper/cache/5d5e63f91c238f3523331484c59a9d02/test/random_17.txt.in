98
480785609 102840440 35318610
901113450 141143261 552980329
895425577 306899196 368400204
59732143 43839203 576024956
53162159 40090503 293670995
207226084 109427567 763738279
995359147 698227127 760098649
234869875 9864708 295925827
394593383 354084921 497117721
390219924 238280103 441519965
719413999 295453254 708951450
936433366 673032512 317299999
864934039 33921949 179243984
104479467 91277831 124779143
929003591 352069889 118776510
86761425 36322977 597045524
914033423 663332452 369385274
13229419 6758556 484884399
929010197 53553531 667122941
730367958 296025352 164090405
985997279 693403764 312273570
340127973 213270780 308072180
721668146 529656398 428185648
464759661 183755553 697992549
444030067 327215005 391160796
830969644 208682324 795858516
571843802 505042528 704116044
848256978 490530924 82059894
286737511 279865824 705980244
467702258 397817416 148988848
896704574 506062344 417278416
536931342 506784942 394306184
301872158 157989258 400986600
561532156 139663015 405084769
765524806 302656525 840790340
115254989 89951940 16866728
224817220 104724320 519195380
50647303 7499550 599013485
406468731 113387394 341446578
576804702 93370011 263396553
228836712 168541477 514981092
176794467 62021808 880095972
404129869 45226808 992898214
365535560 245013307 737340118
876185932 342947520 403798214
232040255 211325815 690942155
584643147 468383769 242884044
236243642 215313430 227881474
134962711 73035755 761839793
499584607 406353853 999169214
554904189 292070643 554904189
207606683 75808809 622820049
698941293 150388681 698941293
4323841 1655297 652899991
799245526 446712657 799245526
56688451 35684927 396819157
318528746 140477129 637057492
839696983 111896086 839696983
921898774 527754021 921898774
185882831 175973184 185882831
716746584 139418854 716746584
981062399 374943341 981062399
692184474 216801409 692184474
963833909 155105546 963833909
887380039 483802867 887380039
192959267 68706534 578877801
531712292 3598046 531712292
701771183 393740026 701771183
69323016 59006043 485261112
789832583 636949352 789832583
813781330 162347710 813781330
540368261 75836102 540368261
904140543 825458715 904140543
1000000000 600000000 200000000
999999936 874999944 652777736
1000000000 750000000 950000000
999999990 399999996 311111108
1000000000 750000000 750000000
999999984 629032248 532258056
999999984 770833321 395833327
999999990 199999998 233333331
999999990 745454538 745454538
1000000000 560000000 360000000
999999963 756097533 707317047
999999976 382352932 367647050
1000000000 100000000 700000000
999999988 884615374 961538450
1000000000 775000000 825000000
999999999 333333333 666666666
1000000000 296875000 890625000
1000000000 200000000 820000000
999999996 666666664 833333330
999999975 743589725 794871775
999999996 916666663 916666663
1000000000 775000000 25000000
999999936 614457792 228915648
1000000000 70000000 310000000
999999968 499999984 295454536
