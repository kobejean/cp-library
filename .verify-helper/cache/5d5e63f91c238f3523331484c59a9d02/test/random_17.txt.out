335096252
91
895425547
43928731
31
207226003
859836635
46
394593368
228917493
97
936433296
862421303
83
929003502
17102652
18
13229363
275001113
16
985997201
194694798
345352856
57
34156098
38283945
74
141376095
-1
35
448352260
200336724
38
29554289
-1
22
11240769
-1
33
192268157
-1
16
36739070
-1
10
46408047
160516828
54
10381726
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
2
63
15
18
3
27
41
12
54
4
6
18
7
23
33
1
21
40
4
28
11
9
41
3
22
